### DISCUSSION TITLE
Lock Screen Ads (LSA)

### DISCUSSION THREAD

**Discussion by irineos1995** [amzn/ads-advanced-tools-docs#147] [Q&A]
Hello community,
I have read and tried the endpoints but wasn't able to get LSA (Lock Screen Ads) through the API.
Is it available in the API and if yes, which endpoint should I use?

Thank you!

**Comment by elizafarley** [comment]
Hi @irineos1995, thanks for your question! Currently the Amazon Ads API does not support LSA. These are only available through the advertising console.

**Reply by irineos1995** [reply]
Thank you for your reply @elizafarley 
So I am guessing the API doesn't provide insights for LSA at all right?
Is there any other way to pull information such as spend, impressions etc. for LSA ads?

Thank you!

**Reply by elizafarley** [reply]
Unfortunately, those metrics are only available via the advertising console right now.
